### DISCUSSION TITLE
升级6.15固件后，r68s的lan口连接的nas无法使用zerotier（群晖套件）

### DISCUSSION THREAD

**Discussion by miantanjun** [coolsnowwolf/lede#9590] [R68S / 电犀牛专区] [answered]
r68s作为主路由，lan口接个nas，两个都分别使用zerotier做内网穿透。
昨天升级6.15那个固件后，r68s的zerotier能在外面正常连接，nas的zerotier（群晖套件版）连接不了，ping没反应
有解决办法吗？
![image](https://user-images.githubusercontent.com/102712739/174007221-16745cc9-b0c3-40fc-b633-5ccc243f5748.png)
![image](https://user-images.githubusercontent.com/102712739/174007288-7bf4f5fe-9e4a-42b7-afd5-0a2190044db3.png)

**Comment by miantanjun** [comment, marked as answer]
加了几条防火墙自定义规则，问题解决了。
![image](https://user-images.githubusercontent.com/102712739/174023964-eecabc52-9615-4d11-b671-355616b32b0d.png)
iptables -I FORWARD -i ztwdjb4d7m -j ACCEPT
iptables -I FORWARD -o ztwdjb4d7m -j ACCEPT
iptables -t nat -I POSTROUTING -o ztwdjb4d7m -j MASQUERADE
把“ztwdjb4d7m”改成自己的zerotier的就行。

**Reply by xiehuijie** [reply]
## 另一种更优雅的解决方案
### 1. 创建Zerotier的接口
使用静态地址，填写在虚拟局域网种设定的ip
![image](https://user-images.githubusercontent.com/45307429/174027802-e9507ddd-77fa-40dd-9898-69c1be406dc1.png)
![image](https://user-images.githubusercontent.com/45307429/174027861-fba9a603-a1cf-4fe8-8bee-50ee338d353f.png)
绑定接口
![image](https://user-images.githubusercontent.com/45307429/174027290-102a395d-359b-4415-8257-69ebf40ca6c5.png)
分配防火墙区域
![image](https://user-images.githubusercontent.com/45307429/174027417-c2188545-d48d-4c84-93d7-816ed6448d39.png)
最后大概是这样
![image](https://user-images.githubusercontent.com/45307429/174027110-e9492e79-b29b-41c9-8598-fa15450f85c2.png)
### 2.配置防火墙规则
将zerotier区域内的均设为允许
![image](https://user-images.githubusercontent.com/45307429/174028098-6f71b51a-db9e-4903-897d-1c5a7f4eed8a.png)
 ### 3.ok了
![image](https://user-images.githubusercontent.com/45307429/174028297-ebd0b142-6e6c-4cbe-8455-bf64f15ea45a.png)
